dist/index.js

```diff
@@ -89,17 +53,15 @@ function getLocation(v) {
 }
 
 function textMessage(v) {
+  let path = getLocation(v);
```

Comment:
nit: (for this and similar lines): I prefer `const` for values that don't need to change, since it _restricts_ the values from changing.

Reply:
This has been updated